model/user/user.go

```diff
@@ -419,6 +419,30 @@ func (u *DBUser) DeleteRoles(roles []string) error {
 	return nil
 }
 
+func (u *DBUser) ImplicitSubscriptionIDs() []string {
```

Comment:
Can we add a function comment to this?

Reply:
Idk if I agree with the wording of "implicit" since users _do_ explicitly subscribe to these. It seems like they're more like General subscriptions or something.